cosmos/airflow/graph.py

```diff
@@ -805,7 +908,23 @@ def build_airflow_graph(  # noqa: C901 TODO: https://github.com/astronomer/astro
             render_config=render_config,
             enable_owner_inheritance=enable_owner_inheritance,
         )
-        test_task = create_airflow_task(test_meta, dag, task_group=task_group)
+        test_task = generate_or_convert_task(
+            node_converters=node_converters,
+            task_meta=test_meta,
+            dbt_project_name=dbt_project_name,
+            dag=dag,
+            task_group=task_group,
+            node=node,
+            resource_type=DbtResourceType.TEST,  # type: ignore
+            execution_mode=execution_mode,
+            task_args=task_args,
+            test_behavior=test_behavior,
+            source_rendering_behavior=source_rendering_behavior,
+            test_indirect_selection=test_indirect_selection,
+            on_warning_callback=on_warning_callback,
+            normalize_task_id=normalize_task_id,
+            detached_from_parent=detached_from_parent,
```

Comment:
Same feedback as: https://github.com/astronomer/astronomer-cosmos/pull/1759/files#r2470518583

Since we are repeating this block 5 times, it feels like we should refactor and review the `generate_or_convert_task` interface to avoid so many arguments. To share the `render_config` may simplify the interface significantly

Reply:
Addressed in https://github.com/astronomer/astronomer-cosmos/pull/2080